apps/src/templates/teacherDashboard/CourseOfferingHelpers.jsx

```diff
@@ -1,5 +1,8 @@
 import i18n from '@cdo/locale';
 
+// The (None) option in editor dropdowns
+export const translatedNoneOption = `(${i18n.none()})`;
```

Comment:
nit: you can decide what's the most reasdable to you here, but I would consider inlining this constant because there is only one of it and it is not providing a mapping from backend values.

Reply:
Good call! Will do